documentation/docs/libraries/nodejs/how_to/_12_mqtt/_expected_output.md

```diff
@@ -0,0 +1,7 @@
+```json
+{
+  topic: 'blocks',
+  payload: '{"protocolVersion":2,"parents":["0x7298db112c825c949ae399b26fa01e6147e916c1dcaff2a2fc487f3beca37f3e","0x7d052b5938882ed514a8419f0daaa567e029fd26bb8384a782fadc73bf4197e1","0xb732a27feb1817bee363efdeb88fa4d3c7ff3ebf44ff72f2e3decc6bf26bfc6e","0xc24af5a4ebdbd4e5be7b0858ca5d3b3333d482c336cd4df3a69a487df31039c6"],"payload":{"type":5,"tag":"0x484f524e4554205370616d6d6572","data":"0x57652061726520616c6c206d616465206f662073746172647573742e0a436f756e743a20353339323036340a54696d657374616d703a20323032322d30392d30325431333a34383a31375a0a54697073656c656374696f6e3a20393131c2b573"},"nonce":"187545"}'
+}
+[...]
```

Comment:
lots of the outputs aren't prettified actually, I did that manually to help the user a bit. I believe in this case the fault lies in the payload being wrapped in `'`  as a string rather than an object.  I've just prettified it.